### PR TITLE
Use o-grid v4

### PR DESCRIPTION
**Pull request by kaelig**
- CSS gets adapted to the new grid
- No markup was harmed during the upgrade

[merge when o-grid v4 is released]

### CODE REVIEW

File: src/scss/base.scss [hunk 1]
```diff
@@ -61,7 +68,7 @@
 	}
 }
 .o-header__inner {
-	@include oGridColumn(12);
+
```
Comment (AlbertoElias) [inline src/scss/base.scss]: why leave an empty selector?

Comment (kaelig) [inline src/scss/base.scss]: Good catch, removed in 3127108

**Comment by AlbertoElias** [comment]
:+1:

**Comment by expe** [comment]
I've merged the ES6 stuff from master and migrated tests to webpack. Seems to be working fine, let me know if I've missed anything.

**Pull request merged**